src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -282,6 +282,34 @@ public void Dispose()
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task Read_Disposed_ReturnsZero(bool transferEncodingChunked)
+        {
```

Comment:
I don't actually think we have any tests for this before. The disabled tests are:
- Read_LargeLengthAsynchronous_Success
- Read_LargeLengthSynchronous_Success
- EndRead_InvalidAsyncResult_ThrowsArgumentException
- EndRead_CalledTwice_ThrowsInvalidOperationException
- Read_FromClosedConnectionAsynchronously_ThrowsHttpListenerException
- Read_FromClosedConnectionSynchronously_ThrowsHttpListenerException